docs/azure-tre-overview/airlock.md

```diff
@@ -141,13 +141,15 @@ graph LR
 
 
 TRE:
+
 * `stalimex` - storage (st) airlock (al) import (im) external (ex)
 * `stalimip` - storage (st) airlock (al) import (im) in-progress (ip)
 * `stalimrej` - storage (st) airlock (al) import (im) rejected (rej)
 * `stalimblocked` - storage (st) airlock (al) import (im) blocked
 * `stalexapp` - storage (st) airlock (al) export (ex) approved (app)
 
-Workspace
+Workspace:
+
```

Comment:
adding the line breaks to fix the bullets, as today all of them are showing as 1 line